extensions/cornerstone/src/services/SegmentationService/SegmentationService.ts

```diff
@@ -960,13 +976,29 @@ class SegmentationService extends PubSubService {
 
     const segmentationId = options?.segmentationId ?? `${csUtils.uuidv4()}`;
 
+    const useSharedArrayBufferValue = window.config?.useSharedArrayBuffer;
```

Comment:
Again, don't repeat the code here, use a shared function that you import from core